autogen.sh

```diff
@@ -728,7 +728,7 @@ _EOF
 
         echo_n "Checking for autoconf version... "
         recreate_tmp
-        ver=2.67
```

Comment:
Is 2.67 not working?

The comment below is more of a distraction now. Might as well remove it.

Reply:
`configure.ac` checks for 2.69, so 2.67 passes `autogen.sh` and will fail at `configure`.